File: src/main/java/com/atividade/plp/questao3/Principal.java
package com.atividade.plp.questao3;

import java.util.Scanner;

public class Principal {

	public static void main(String[] args) {
		Calculo calculo = new Calculo();
		Scanner scanner = new Scanner(System.in);

		System.out.println("Entre com a largura: ");
		double larguraResultado = scanner.nextDouble();
		calculo.setLargura(larguraResultado);
		
		System.out.println("Entre com a altura: ");
		double alturaResultado = scanner.nextDouble();
		calculo.setAltura(alturaResultado);

		System.out.println(calculo.area());
		System.out.println(calculo.diagonal());
		System.out.println(calculo.perimetro());	
		
		scanner.close();
	}

}

File: python/Questao3/Principal.py
from python.Questao3.Calculo import CalculoRetangulo

class Principal():
    largura_resultado = 0
    altura_resultado = 0

    print("Entre com a altura: ")
    altura = input()
    CalculoRetangulo.set_altura(largura_resultado)

    print("Entre com a largura: ")
    largura = input()
    CalculoRetangulo.set_largura(altura_resultado)

    print(CalculoRetangulo.area)
    print(CalculoRetangulo.perimetro)
    print(CalculoRetangulo)
    